.circleci/config.yml

```diff
@@ -121,8 +121,8 @@ commands:
             if [ -z "${NVM_DIR}" ] ; then
               echo 'Appending nvm source string to BASH_ENV'
               echo 'export NVM_DIR="$HOME/.nvm"' >> $BASH_ENV
-              echo '[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"' >> $BASH_ENV
             fi
+            echo '[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"' >> $BASH_ENV
```

Comment:
From what I can tell, this diff is only moving the line to outside the conditional, right? Does that mean that currently the conditional actually fails? If that's the case, I wonder if we need to improve the conditional or revise it?

Reply:
Yes, it's only moving the line outside, in theory, the `[ -z "${NVM_DIR}" ]` condition is not required, as it's described in [NVM readme](https://github.com/nvm-sh/nvm/tree/v0.37.0#install--update-script). However, when I introduced these lines in [this PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/3761/files), I added the condition to prevent path conflicts in case NVM was already installed.

For the line moved, I originally thought that if `$NVM_DIR` is defined, it implied that NVM would be accessible along the workflow, however, this issue uncovered that it's possible to have NVM installed but not loaded at the time when the commands of the Bash environment (`$BASH_ENV`) are executed. For this reason, I decided to move it outside.

Reply:
Thanks for the context Carlos! I see, it makes sense to me.

Nit, now that the conditional only has the $NVM_DIR definition clause, the inline comment (`Appending nvm source string to BASH_ENV'`) doesn't read accurate anymore. Want to change it to something like "Default NVM_DIR to .nvm in home" or something?

Reply:
> Nit, now that the conditional only has the $NVM_DIR definition clause, the inline comment (Appending nvm source string to BASH_ENV') doesn't read accurate anymore. Want to change it to something like "Default NVM_DIR to .nvm in home" or something?

Good point, that message should now only reference the `NVM_DIR`, I'll change it to "Default NVM_DIR to .nvm in home".

Reply:
This suggestion was added in [this commit](https://github.com/wordpress-mobile/gutenberg-mobile/pull/3990/commits/24adc32c8e143f7715d0112fdcaa6abe1ab4b663).